app/admin/dashboard.rb

```diff
@@ -3,31 +3,21 @@
   menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
 
   content title: proc{ I18n.t("active_admin.dashboard") } do
-    div class: "blank_slate_container", id: "dashboard_default_message" do
-      span class: "blank_slate" do
-        span I18n.t("active_admin.dashboard_welcome.welcome")
-        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+
+    columns do
+      column do
+        panel "Info" do
+            @user_chapter = Chapter.where(id: current_admin_user.chapter_id)
+            @user_chapter.each do |c|
+                if c.is_active?
```

Comment:
@kexline4710 now, does this work okay since it's boolean? I was debating leaving just c.is_active or this...didn't want to get in the fun 3-state boolean stuff I was reading about....thanks for zee help!